sidecar/interfaces.go

```diff
@@ -252,6 +274,16 @@ func (t *Ticket) OfferDigest() ([32]byte, error) {
 			return result, err
 		}
 
+	case VersionUnannouncedZeroConf:
+		err := lnwire.WriteElements(
```

Comment:
In the future, we can switch to a TLV format here to make changes like this a bit easier, but totally not blocking atm.